app/templates/buyers/brief_publish_confirmation.html

```diff
@@ -0,0 +1,64 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Your account - Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+    {%
+        with items = [
+        {
+            "link": "/",
+            "label": "Digital Marketplace"
+        }
+        ]
+    %}
+    {% include "toolkit/breadcrumb.html" %}
+    {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+<div class='grid-row'>
+  <div class='column-two-thirds large-paragraph'>
+    {% with
+        heading = "Publish your requirements and evaluation criteria",
+        smaller = True,
+        with_breadcrumb = True
+    %}
+        {% include "toolkit/page-heading.html" %}
+    {% endwith %}
+    <p class="padding-bottom-small">All requirements are published on the Digital Marketplace where anyone can see them.</p>
```

Comment:
There is a new secret style `dmspeak` in the toolkit that I think if you put it on the wrapper div will mean we don't need the `padding-bottom-small`s on each paragraph.